Sources/Plasma/PubUtilLib/plModifier/plImageLibMod.h

```diff
@@ -75,6 +75,9 @@ class plImageLibMod : public plSingleModifier
     };
 
     size_t  GetNumImages() const { return fImages.size(); }
+    plBitmap* GetImage(const ST::string&) const;
+    const std::vector<plBitmap*> GetImages() const;
```

Comment:
```suggestion
    std::vector<plBitmap*> GetImages() const { return fImages; }
```

This will copy the contents of the container for you.